Bottom: fd7f40bfb83878bc362c31332b01602f41a00b85
Top:    da51d132eccae92df38d64340105cf668fe27c00
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-05-01 23:24:18 +0100

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a5fa8d3..37a47cc 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -324,12 +324,12 @@ object3d_from_contours (const POLYAREA *contours,
     }
 
     faces[npoints] = make_face3d (); /* bottom_face */
-    face3d_set_normal (faces[npoints], 0., 0., 1.);
+    face3d_set_normal (faces[npoints], 0., 0., -1.);
     face3d_set_appearance (faces[npoints], top_bot_appearance);
     object3d_add_face (object, faces[npoints]);
 
     faces[npoints + 1] = make_face3d (); /* top_face */
-    face3d_set_normal (faces[npoints + 1], 0., 0., -1.);
+    face3d_set_normal (faces[npoints + 1], 0., 0., 1.);
     face3d_set_appearance (faces[npoints + 1], top_bot_appearance);
     object3d_add_face (object, faces[npoints + 1]);
